### PR TITLE
installed and configed hydra-role-management gem

### PR DESCRIPTION
**Pull request by mikedaul**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -48,7 +52,15 @@ def find_or_create_system_user(user_key)
     end
 
     def curators
-      RoleMapper.whois(CURATOR_GROUP)
+
+      #RoleMapper.whois(CURATOR_GROUP)
+
+      if curator_group = Role.where(name: User::CURATOR_GROUP).first
```
Comment (dchandekstark) [inline app/models/user.rb]: I assume you can use here instead:

```ruby
Role.find_by(name: User::CURATOR_GROUP)
```

File: app/models/ability.rb [hunk 2]
```diff
@@ -6,9 +6,28 @@ class Ability
...
+    #
+    # if user_groups.include? 'special_group'
+    #   can [:create], ActiveFedora::Base
+    # end
+
+    if current_user.admin?
+      can [:create, :show, :add_user, :remove_user, :index, :edit, :update, :destroy], Role
+    end
+
+    # from 'old' role management
```
Comment (coblej) [inline app/models/ability.rb]: I would remove this comment.

File: app/models/user.rb (hunk 1 above)
Comment (mikedaul) [reply app/models/user.rb]: Thanks David - Jim originally suggested something like that, but it ended up failing a bunch of tests. I think the rationale for where we ended up was accounting for the possibility that the curator group may not exist.

Comment (dchandekstark) [reply app/models/user.rb]: If the curator group does not exist `find_by` should return `nil`.  `find_by!` (bang) raises an exception.

**Comment by mikedaul** [comment]
Interesting – thanks! I don’t know enough about it to really have an opinion.

FWIW when I tried changing that chunk to what you suggested, it resulted in something like 66 failed tests ☹

File: app/models/user.rb (hunk 1 above)
Comment (mikedaul) [reply app/models/user.rb]: Turns our I just misunderstood your suggestion - seems to work great, actually :)

Thanks again!

**Pull request merged**